app/ts/components/simulationExplaining/Transactions.tsx

```diff
@@ -170,6 +173,83 @@ export function Transaction(param: TransactionVisualizationParameters) {
 	)
 }
 
+type TransactionOrSignedMessageWithBlockTimeManipulatorParams = {
+	simulationAndVisualisationResults: SimulationAndVisualisationResults
+	simTx: SimulatedAndVisualizedTransaction | VisualizedPersonalSignRequest
+	renameAddressCallBack: RenameAddressCallBack
+	editEnsNamedHashCallBack: EditEnsNamedHashCallBack
+	removeTransactionOrSignedMessage: (transactionOrMessageIdentifier: TransactionOrMessageIdentifier) => void
+	activeAddress: bigint
+	addressMetaData: readonly AddressBookEntry[]
+	blockTimeManipulation: BlockTimeManipulationWithNoDelay
+}
+
+const TransactionOrSignedMessageWithBlockTimeManipulator = ({ simTx, renameAddressCallBack, editEnsNamedHashCallBack, removeTransactionOrSignedMessage, simulationAndVisualisationResults, activeAddress, addressMetaData, blockTimeManipulation }: TransactionOrSignedMessageWithBlockTimeManipulatorParams) => {
```

Comment:
`SignedMessage` was changed to `Message` above, should this also be `TransactionOrMessageWith...` instead (same for the params)?